### PR TITLE
push image to docker hub

### PR DESCRIPTION
**Pull request by Danilosimoes**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,4 +5,11 @@ build/image:
 	@echo "Building image u-vocab-api:${VERSION}"
 	@echo " "
 	./gradlew dockerBuildImage --parallel -Pversion=${VERSION}
-	docker tag u-vocab-api:${VERSION} uvocab/u-vocab-api:${VERSION}
\ No newline at end of file
+	docker tag u-vocab-api:${VERSION} uvocab/u-vocab-api:${VERSION}
+
+login/push:
```
Comment (robsonoduarte) [inline Makefile:10]: to standardize the names of "task" on makefile my suggestion for this is `image/push`

File: build.gradle [hunk 2]
```diff
@@ -91,6 +91,10 @@ spotless {
 docker {
 	springBootApplication {
 		baseImage = 'azul/zulu-openjdk:20.0.2-20.32.11'
+		registryCredentials {
+			username = System.getenv('DOCKER_LOGIN')
+			password = System.getenv('DOCKER_PASSWORD')
+		}
```
Comment (robsonoduarte) [inline build.gradle:97]: this is not necessary, already we do the login in docker hub by shell command.

File: Makefile (hunk 1 above)
Comment (Danilosimoes) [reply Makefile:10]: Ok!! thanks, I'll change

**Review by robsonoduarte — APPROVED** [review]
great job :rocket: :guitar:

**Pull request merged**